schema/core-im/core-im-source.yaml

```diff
@@ -56,6 +56,7 @@ $defs:
           fields in the data itself. Extensions are not expected to be natively understood, 
           but may be used for pre-negotiated exchange of message attributes between systems.
     heritableRequired:
+      - id
```

Comment:
@mbrush Alex and I didn't think we needed `id` required at the top `Entity` level. We thought it made sense to require `InformationEntity` to require `id` but not necessarily every `Entity` everywhere. Also, if an ancestor class requires an attribute it does not need to be respecified on the descendant class.

Reply:
However, now that you are making me think about this. I think all Entity's should require an id and type. I approve this change. I may remove the redundant requirement on InformationEntity since it is not needed at that level.